source/sass/2017/markdown/code.scss

```diff
@@ -6,6 +15,7 @@
 .MarkdownBody pre,
 .MarkdownBody code {
   font-family: $monospace-font;
+  word-break: break-word;
```

Comment:
I do not think it works. 
![image](https://user-images.githubusercontent.com/22890026/78608082-14648600-7815-11ea-9868-f0d9a7fd6faf.png)

Reply:
It don't solve `<pre>` but inline `<code>`.

before:

![before](https://user-images.githubusercontent.com/11372753/78615631-aa93bd00-78a4-11ea-89f4-25172a367d63.png)

after:

![after](https://user-images.githubusercontent.com/11372753/78615677-c008e700-78a4-11ea-8c47-54a1ff722851.png)

The same property also works in `<a>`.

Reply:
That's really great. The use of inline code in the code block is out of my expectation. I do not think of this practice.

Reply:
It seems that you address part of the problem. As you can see, the last line have a word break since it is an inline code. However, the first two do not. One is in pre>code and the other one is ul.
![image](https://user-images.githubusercontent.com/22890026/78633232-ea7f8380-7855-11ea-9260-bef6dd5c4d26.png)

Reply:
Here is the markdown source code
```bash
# to debugto debugto debugto debugto debugto debugto debugto debugto debugto debugto debugto debugto debugto debugto debug
r2 -d
```
* 123123123123123123123123123123123123123123123123123
*  789 `print('hello world')safsdfasfasdfsadfasdfdsaf`

Reply:
I think the no word wrap for the code is fine.

> You can't just go wrapping code without some indication that it was wrapped." - one might as well say, "You can't just go cutting code off the right hand side of the box without some indication it was cut off.

Reply:
Maybe I found the source of the quotation:

<https://meta.stackexchange.com/questions/222429/why-dont-code-blocks-in-stack-exchange-use-word-wrap-by-default>

The page mainly talked about the word warp of CODE BLOCKS, not inline code. 

I think there's no need to use word warp in code blocks, but inline code needs mainly for visual experience. In practice, inline code mainly describes short code, path, variable and so on, in which path may be long enough to wrap, and if one use inline code to describe them, (s)he knows what it means. 

I found some examples of using wrap in inline code:

![so](https://user-images.githubusercontent.com/11372753/78674829-c9cb3280-7916-11ea-9673-53de15967dc0.png)

(Source: <https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression?rq=1>)

![se](https://user-images.githubusercontent.com/11372753/78675192-49590180-7917-11ea-9f04-8cf32bfda73b.png)

(Source: <https://meta.stackexchange.com/questions/222429/why-dont-code-blocks-in-stack-exchange-use-word-wrap-by-default>)

![mdn](https://user-images.githubusercontent.com/11372753/78674975-f8e1a400-7916-11ea-8341-5af81f91a3cd.png)

(MDN, using Chrome Developer Tool to add words in `<code>`)

However, the quotation is right in some ways: indication is important. Using rounded rectangle in inline code may be a better idea. I tried it just for trying and think that's good.

![a_better_idea](https://user-images.githubusercontent.com/11372753/78675409-9b9a2280-7917-11ea-8eb0-21f17299328a.png)

I'll also fix the code.

Reply:
Thank you for the contribution and sorry for the confusion. I mean no need for word wrap for code block. It is necessary for inline code

Reply:
For the inline code design, I recommend addressing the following issues.
* Big round corner is attractive, but the design of the whole website is square with little round corner. Thus the big round corner of the inline code breaks the whole style of the website. This is unacceptable.
* To create the visual difference between inline code and body, the background color should have a higher contrast ratio.
We can follow other website's inline code design, like StackExchange